ci/infra/testrunner/skuba/skuba.py

```diff
@@ -81,8 +91,8 @@ def node_bootstrap(self):
         self._verify_bootstrap_dependency()
 
         master0_ip = self.platform.get_nodes_ipaddrs("master")[0]
-        cmd = "node bootstrap --user {username} --sudo --target \
-                 {ip} my-master-0".format(ip=master0_ip, username=self.conf.nodeuser)
+        cmd = "{} node bootstrap --user {username} --sudo --target \
+                 {ip} my-master-0".format(self.SKUBA_VERBOSE, ip=master0_ip, username=self.conf.nodeuser)
```

Comment:
Instead of doing this, pass it on the `env` dictionary (I don't know if it's exposed for this funcion). Just saying because doing this failed miserably in `skuba-update`.